region.go

```diff
@@ -47,8 +55,7 @@ type Region struct {
 	Description string
 	VPCRegion   string
 	COSRegion   string
-	Zones       []string
-	SysTypes    []string
+	Zones       map[string]SysTypes
 	VPCZones    []string
 }
 
```

Comment:
Considering so many breaking changes we made to this Region variable, can we make this variable private instead of exposing this to outside world?

Reply:
Done. I checked if it impacts any other repos, it doesn't